cmd/release/main.go

```diff
@@ -179,17 +190,30 @@ type config struct {
 }
 
 type runner struct {
+	builder  builder
 	releaser releaser
 	tagger   tagger
 	config   config
 }
 
+// TODO sha512 the artifacts!
+
 func (r runner) run() error {
 	fmt.Print("tagging repository ")
 	if err := r.tagger.tag(r.config.version); err != nil {
 		return err
 	}
 	fmt.Println("🐲")
+	fmt.Print("checking out tag ")
+	if err := r.tagger.checkout(r.config.version); err != nil {
+		return err
+	}
+	fmt.Println("🐲")
+	fmt.Print("building artifacts ")
```

Comment:
Same as above, might be useful to know which artifacts

Reply:
it will iterate through the uploaded files later in the program...but this is a wrapper for `make release artifacts`.

would the output of the wrapped command be sufficient?